src/Repository/TaskRepository.php

```diff
@@ -125,6 +125,12 @@ private function getTableQueryBuilder(string $searchValue, TaskTableContext $con
             }
         }
 
+        if (null !== $context->filters->status) {
```

Comment:
No getter?

Reply:
No it's just a dto, getters are overkill